### PR TITLE
Add parameters to run specific aqa-tests releases

### PR DESCRIPTION
**Pull request by llxia**
- Add USE_TESTENV_PROPERTIES
- Remove `ADOPTOPENJDK${release}_REPO` logic as it is not used and it complicates the logic for consuming user-provided `ADOPTOPENJDK_REPO` and `ADOPTOPENJDK_BRANCH`

Related: runtimes/infrastructure/issues/6390 and https://github.com/adoptium/aqa-tests/issues/2347

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-Personal/81

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 1]
```diff
@@ -413,6 +414,9 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
             SHAS['OPENJ9'] = "origin/pr/${params.ghprbPullId}/merge"
         }
 
+        USE_TESTENV_PROPERTIES = params.USE_TESTENV_PROPERTIES ? params.USE_TESTENV_PROPERTIES : false
```
Comment (AdamBrousseau) [inline buildenv/jenkins/common/pipeline-functions.groovy:417]: Can simplify with elvis operator
```
USE_TESTENV_PROPERTIES = params.USE_TESTENV_PROPERTIES ?: false
```

**Comment by llxia** [comment]
|Parameters| Build| Comment|
|--|--|--|
| ADOPTOPENJDK_REPO=https://github.com/llxia/openjdk-tests.git<br> ADOPTOPENJDK_BRANCH=test2<br> USE_TESTENV_PROPERTIES=true |https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-Personal/91| use testenv.properties|
|ADOPTOPENJDK_REPO=`empty`<br> ADOPTOPENJDK_BRANCH=`empty`<br> USE_TESTENV_PROPERTIES=false|https://openj9-jenkins.osuosl.org/job/Pipeline-Build-Test-Personal/92| default values|

**Comment by llxia** [comment]
Please note, once `USE_TESTENV_PROPERTIES ` is set to true, the test pipeline will use the values that are defined in testenv.properties (except ADOPTOPENJDK_REPO and ADOPTOPENJDK_BRANCH).

**Review by pshipton — APPROVED** [review]

**Pull request merged**